sound/soc/sof/intel/hda-codec.c

```diff
@@ -155,7 +152,7 @@ int hda_codec_probe_bus(struct snd_sof_dev *sdev)
 }
 EXPORT_SYMBOL(hda_codec_probe_bus);
 
-#if IS_ENABLED(CONFIG_SND_SOC_HDAC_HDMI)
+#if IS_ENABLED(CONFIG_SND_HDA_CODEC_HDMI) || IS_ENABLED(CONFIG_SND_SOC_HDAC_HDMI)
```

Comment:
why do you still care about SOC_HDAC_HDMI?

Reply:
> @libinyang and @xiulipan : can you check the thread on https://mailman.alsa-project.org/pipermail/alsa-devel/2019-August/154561.html about the dyn-pcm-assign mode in patch_hdmi.c.
> 
> It would seem we are going towards DP-MST without the backup PCMs. Can you recall any critical problems with this approach (i.e. reasons for originally adding the backup PCMs to DP-MST support).

@kv2019i , for the backup PCMs is required from pulseaudio for the compability.

Reply:
@plbossart Unless we change all machine drivers at once, we need to keep hdac-hdmi support in the wrapper layer still. Plus this is a handy way to test we are not breaking machine drivers for SST.